src/Symfony/Component/Validator/Constraints/GroupSequence.php

```diff
@@ -12,66 +12,44 @@
 namespace Symfony\Component\Validator\Constraints;
 
 /**
- * A sequence of validation groups.
+ * The GroupSequence class represents a sequence of validation groups.
+ * It is used to enforce a specific order in which validation groups should be processed.
  *
- * When validating a group sequence, each group will only be validated if all
- * of the previous groups in the sequence succeeded. For example:
+ * When validating a group sequence, each group is validated sequentially. A group is only validated if all
+ * previous groups in the sequence succeeded. This approach is beneficial for scenarios where certain validation
+ * groups are more resource-intensive or rely on the success of prior validations.
  *
- *     $validator->validate($address, null, new GroupSequence(['Basic', 'Strict']));
+ * Group sequences can also be used to override the "Default" validation group for a class. When a class has an
+ * associated group sequence and is validated in the "Default" group, the group sequence is applied instead.
  *
- * In the first step, all constraints that belong to the group "Basic" will be
- * validated. If none of the constraints fail, the validator will then validate
- * the constraints in group "Strict". This is useful, for example, if "Strict"
- * contains expensive checks that require a lot of CPU or slow, external
- * services. You usually don't want to run expensive checks if any of the cheap
- * checks fail.
- *
- * When adding metadata to a class, you can override the "Default" group of
- * that class with a group sequence:
- *     #[GroupSequence(['Address', 'Strict'])]
- *     class Address
- *     {
- *         // ...
- *     }
- *
- * Whenever you validate that object in the "Default" group, the group sequence
- * will be validated:
- *
- *     $validator->validate($address);
- *
- * If you want to execute the constraints of the "Default" group for a class
- * with an overridden default group, pass the class name as group name instead:
- *
- *     $validator->validate($address, null, "Address")
```

Comment:
As we have it, I would not remove the example, more docs are always better to me

Reply:
okay, added.